classes/image_types_tegra.bbclass

```diff
@@ -587,23 +615,34 @@ oe_make_bup_payload_common() {
     rm -f ./slot_metadata.bin
     cp ${STAGING_DATADIR}/tegraflash/slot_metadata.bin ./
     mkdir ./rollback
-    ln -sf ${STAGING_BINDIR_NATIVE}/tegra186-flash/rollback_parser.py ./rollback/
+    ln -sf ${STAGING_BINDIR_NATIVE}/tegra186-flash/rollback/rollback_parser.py ./rollback/
```

Comment:
See my comment below about where this gets installed - if that gets changed, this should get changed back to the original